### PR TITLE
Es6 projects common modules onward breaking news

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

ES6ify and Jestify `breaking-news.js`

I have prioritised this module as the tests were previously very non-deterministic and prone to intermittent failure. Jest helps with this by allowing us to mock timer functions (such as `setTimeout`), removing the likelihood of race conditions. 

## What is the value of this and can you measure success?

Shiny things and less breaking builds

## Does this affect other platforms - Amp, Apps, etc?

No

## Tested in CODE?

No

### CODE REVIEW

**Comment by philwills** [comment]
This looks like it will be tremendously useful as a reference for how to do some less trivial testing with Jest.

The `Alert` type in many of the signatures makes things much clearer.

File: static/src/javascripts/projects/common/modules/onward/breaking-news.js [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+// show an alert
+const showAlert = (alert: Alert): Alert => {
+    if (alert) {
+        const $body = bonzo(document.body);
+        const $breakingNews = bonzo(qwery('.js-breaking-news-placeholder'));
+
+        // if its the first time we've seen this alert, we wait 3 secs to show it
+        // otherwise we show it immediately
+        const alertDelay = has(knownAlertIDs, alert.id) ? 0 : DEFAULT_DELAY;
```
Comment (GHaberis) [inline static/src/javascripts/projects/common/modules/onward/breaking-news.js]: Assuming we're using 'has' to check if the specified property name (alert.id) exists as a direct property of `knownAlertIds`, we could just use `hasOwnProperty` instead and drop the 'has' lodash lib? eg.

`const alertDelay = knownAlertIDs.hasOwnProperty(alert.id)  ? 0 : DEFAULT_DELAY;`

Alternatively if we're just checking to see if it has the property at all (direct or inherited) we could just check whether knownAlertIDs[alert.id] is truthy?

File: static/src/javascripts/projects/common/modules/onward/breaking-news.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+const updateKnownAlertID = (id: string, state: boolean): void => {
+    knownAlertIDs[id] = state;
+    storeKnownAlertIDs();
+};
+
+const markAlertAsSeen = (id: string): void => {
+    updateKnownAlertID(id, false);
+};
+
+const markAlertAsDismissed = (id): void => {
```
Comment (GHaberis) [inline static/src/javascripts/projects/common/modules/onward/breaking-news.js]: Looks like you missed the type of id here

**Review by GHaberis — APPROVED** [review]
Looks good to me 👍

**Pull request merged**